source/frontend/src/features/mapSideBar/management/tabs/ManagementFileTabs.test.tsx

```diff
@@ -177,7 +197,7 @@ describe('ManagementFileTabs component', () => {
   });
 
   it('notes tab can be changed to', async () => {
-    const { getByRole } = await setup(
+    const { getByRole, baseElement, findByRole } = await setup(
```

Comment:
nit: I don't think this test is using either of **baseElement** or **findByRole** - please remove